about/aml-kyc-requirements.md

```diff
@@ -0,0 +1,20 @@
+3. The Foundation will run a Sanctions Screening and Fraud Monitoring on each recipient of its funds.  As a condition of receiving the funds you represent to us, now and until the latter of the submission of a report on the status of the work covered by the proposal or the use of all of the funds, (i) that you are not in violation of any law relating to terrorism or money laundering (“Anti-Terrorism Laws”), including Executive Order No. 13224 on Terrorist Financing, effective September 24, 2001 (the “Executive Order”), and the Uniting and Strengthening America by Providing Appropriate Tools Required to Intercept and Obstruct Terrorism Act of 2001(Title III of P.L. No. 107-56) (known as the “PATRIOT Act”).  (ii)  neither you or any affiliated person or entity is a person that is listed in the annex to, or is otherwise subject to the provisions of, the Executive Order or a person that is named as a “specially designated national and blocked person” on the most current list published by the US Department of the Treasury, Office of Foreign Assets Control (“OFAC”) at its official website or any replacement website or other replacement official publication of such list; (iii) neither you or any affiliated person or entity is subject to blocking provisions or otherwise a target of sanctions imposed under any sanctions program administered by OFAC; and (iv) neither you or any affiliate person or entity deals in, or otherwise engages in any transaction relating to any property or interests in property blocked pursuant to the Executive Order.
+4. With certain limited exceptions, in the following January the Zcash Foundation will report the value of the funds as taxable income on either US tax form 1099-MISC (for US taxpayers) or 1042-S (for foreign persons). These forms will report the value of the award in USD at the date it was distributed. You may need to include this income when filing your taxes, and it may affect your total tax due and estimated tax payments. Here are more details on [filing the 1099-MISC](https://www.irs.gov/forms-pubs/about-form-1099-misc) in the US, and its tax implications.
+
+_Disbursement_: Your funds will be disbursed in a shielded Zcash cryptocurrency (ZEC), then it will be via an on-blockchain fund transfer transaction. The Foundation will use this third-party service and market for converting ZEC to other currencies are listed here based on the price of the agreed-upon date close of day at : [https://messari.io/asset/zcash](https://messari.io/asset/zcash). For all grants, the agreed-upon date will be the date that the grant was approved, as noted in the grant platform. Note that the Zcash Foundation understands the regulatory and compliance risks associated with transacting in cryptocurrencies. and 
+
```

Comment:
hanging "and" after the last sentence. Maybe incomplete?

Reply:
good catch. removed.